src/praisonai-agents/tests/unit/lsp/test_lsp.py

```
@@ -323,5 +325,87 @@ def test_client_not_running_initially(self):
+        # Walk may still hit a real marker above tmp; assert it's not the file dir.
+        root = detect_root_uri(str(f))
+        assert root != f"file://{f.parent}"
+
+
+class TestProbe:
+    """Tests for probe() availability check."""
+
+    def test_available_when_on_path(self, monkeypatch):
+        import praisonaiagents.lsp.config as cfg
+        monkeypatch.setattr(cfg.shutil, "which", lambda cmd: "/usr/bin/" + cmd)
+        available, command, hint = probe("python")
```

Ruff (0.16.1)

[warning] Unpacked variable `hint` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)